### DISCUSSION TITLE
"waiting for verification" cleanup

### DISCUSSION THREAD

**Discussion by andupotorac** [supabase/supabase#29742] [Questions]
What's the best way to clean-up the users from the Authentication table, who've been in "waiting for verification" mode for a while? In general it might be users who used random emails that don't exist.

**Comment by GaryAustin1** [comment]
Setting up a cron task to run every day or week would probably be easiest.  Then delete with a where clause involving created_at being more than x days/weeks ago and confirmed_at being null.  You can see all the columns you can use with the table UI on auth.users.

**Reply by andupotorac** [reply]
So there is no automated way of doing this?

**Reply by GaryAustin1** [reply]
Not from Supabase that I know of.

**Reply by GaryAustin1** [reply]
The UI features do change all the time though, just don't recall anything about that or seen it myself.

**Reply by andupotorac** [reply]
That, and the ability to batch edit those users too, feels like a must have. What are people just going to do with all the test@test.com emails
